docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx

```diff
@@ -13,8 +12,24 @@ Enabling [Preact](https://preactjs.com/) in your FastStore project can reduce th
 
 ## Before you begin
 
+Before implementing Preact, take the following considerations:
+
+<Steps>
+
+### Update `@faststore/cli` package
+
 Make sure the `@faststore/cli` package from your project is updated to the latest version. To do so, follow the instructions in [Updating the '@fastore/cli' package version](https://developers.vtex.com/docs/guides/faststore/project-structure-updating-the-cli-package-version).
 
+### Understand the experimental nature of Preact
+
+Preact is currently an experimental feature, which may have limitations and could cause issues with some store customizations.
+
+- Before using it in the production environment, make sure to test it to ensure compatibility with your store.
+
+- Preact is a lightweight alternative to React, with reduced feature support. This may lead to compatibility issues with advanced React APIs and some third-party libraries.
```

Comment:
```suggestion
- Preact is a lightweight alternative to [React](https://react.dev/), with reduced feature support. This may lead to compatibility issues with advanced React APIs and some third-party libraries.
```